tests/arp/test_wr_arp.py

```diff
@@ -57,7 +57,7 @@ def setupFerret(self, duthost, ptfhost):
         ptfhost.copy(src="arp/files/ferret.py", dest="/opt")
 
         result = duthost.shell(
-            cmd='''ip route show proto zebra type unicast |
+            cmd='''ip route show proto 186 type unicast |
```

Comment:
I checked once more on 201911 and the situation is now reversed. I can see why this change is required for 201911. @pavel-shirshov are we missing def for protocols on 201911 image.

Reply:
@vsenchyshyn  Some fixes were lost during migration to pythes version. See https://github.com/Azure/sonic-mgmt/pull/1354

with this cmd `ip route show type unicast | sed -e '/proto 186\|proto zebra/!d' -e '/default/d' -ne '/0\//p' | head -n 1 | sed -ne 's/0\/.*$/1/p'` all sonic version should be supported

Reply:
I see. Thanks for the clarification.

Can we then use the new command from Ansible in here instead of limiting it to proto 186 only as this will not work on 201811 image.